src/types/index.ts

```diff
@@ -503,3 +503,15 @@ export type CoinTypeAddress = {
 
 export const IBC_TRANSFER_TYPE = "/ibc.applications.transfer.v1.MsgTransfer";
 export const WASM_TYPE = "/cosmwasm.wasm.v1.MsgExecuteContract";
+
+export type TokenBalance = {
+  symbol: string;
+  address: string;
+  decimals: number;
+  balanceInDecimal: string;
```

Comment:
Rename this variable to balance, and store it in wei not in decimals, the format must be done outside on the implementation